TD tooltip fixes

clemty opened the pull request:
- Mention Apache being strong vs aircraft.
- Mention engineers will damage buildings, not just capture them.
- Removed whitespace before line breaks

reverted:
- Sentences are now (mostly) lowercase, end with full-stop.
- Changed "defence" tab tooltip to "defense".

abcdefg30 commented on mods/cnc/rules/aircraft.yaml:
```diff
@@ -47,7 +47,7 @@ HELI:
 	Tooltip:
 		Name: Apache Longbow
-		Description: Helicopter Gunship with Chainguns.\n  Strong vs Infantry, Light Vehicles\n  Weak vs Tanks
+		Description: Helicopter gunship with chainguns.\n  Strong vs infantry, light vehicles &\n  aircraft.\n  Weak vs tanks.
```
`Strong vs Infantry, Light vehicles, Aircraft\n  Weak vs Tanks`, to be consistent with #8277 and #7967, please.

abcdefg30 commented on mods/cnc/rules/aircraft.yaml:
```diff
@@ -4,7 +4,7 @@ TRAN:
 	Tooltip:
 		Name: Chinook Transport
-		Description: Fast Infantry Transport Helicopter.\n  Unarmed
+		Description: Fast infantry transport helicopter.\n  Unarmed.
```
No `.` after `Unarmed`, please.

abcdefg30 commented on mods/cnc/rules/aircraft.yaml:
```diff
@@ -98,7 +98,7 @@ ORCA:
 	Tooltip:
 		Name: Orca
-		Description: Helicopter Gunship with AG Missiles.\n  Strong vs Buildings, Tanks\n  Weak vs Infantry
+		Description: Helicopter gunship with AG missiles.\n  Strong vs buildings, tanks.\n  Weak vs infantry.
```
`Strong vs Buildings, Tanks\n  Weak vs Infantry`

abcdefg30 commented:
Above affects all `Strong/Weak vs`. Sorry for making you revert that.
Otherwise we would get inconsistent with the other mods.
(Or do we want `TD` to be special again?)

GraionDilach commented:
Shouldn't the second tab be named as Support? Or do I remember wrong that it was treat as such...

clemty commented:
>Above affects all Strong/Weak vs. Sorry for making you revert that.
>Otherwise we would get inconsistent with the other mods.
>(Or do we want TD to be special again?)

I didn't intend to make spelling inconsistent with the other mods, it's just that the current way of capitalizing some units/unit types seemed incorrect to me

Making unit types upper-case in the strong/weak sections creates another inconsistency:
- Fast infantry armed with grenades. / Anti-tank/Anti-aircraft infantry. ... Weak vs Infantry

also:
- Strong vs Buildings, slow-moving targets

why "Buildings" but not "Targets" ?

>No . after Unarmed, please.

This would create a case where most lines end in "." and only "Unarmed" does not - according to http://www.edufind.com/english-grammar/period-full-stop-or-point/ it is ok to put "." after a single word.

>Shouldn't the second tab be named as Support? Or do I remember wrong that it was treated as such...

That's a good point - the Tiberium silo is not a defense structure

Phrohdoh commented:
A description for an actor is not a sentence and therefor, technically, does not need to end with a full-stop.

clemty commented:
updated

GraionDilach commented on mods/cnc/rules/aircraft.yaml:
```diff
@@ -47,7 +47,7 @@ HELI:
 	Tooltip:
 		Name: Apache Longbow
-		Description: Helicopter Gunship with Chainguns.\n  Strong vs Infantry, Light Vehicles\n  Weak vs Tanks
+		Description: Helicopter Gunship with Chainguns.\n  Strong vs Infantry, Light Vehicles &\n  Aircraft\n  Weak vs Tanks
```
I don't really like the ampersand here.

abcdefg30 commented on mods/cnc/rules/infantry.yaml:
```diff
@@ -136,7 +136,7 @@ E6:
 	Tooltip:
 		Name: Engineer
-		Description: Infiltrates and captures enemy structures.\n  Strong vs Nothing\n  Weak vs Everything
+		Description: Infiltrates and captures enemy structures.\n  Strong vs Buildings\n  Weak vs Everything
```
`Infiltrates and captures enemy structures.\n  Unarmed`

abcdefg30 commented on mods/cnc/rules/structures.yaml:
```diff
@@ -914,7 +914,7 @@ BRIK:
 	Tooltip:
 		Name: Concrete Barrier
-		Description: Stop units.
+		Description: Stops infantry & most tanks.
```
`&` -> `and`, please.

clemty commented:
updated.

>I don't really like the ampersand here.

Neither do I, but it is used that way in vehicles.yaml (4 hits) and structures.yaml (5 hits).

>& -> and, please.

I changed all occurrences of "&" into "and"

abcdefg30 commented:
Thanks. :+1:

Unit158 commented on mods/cnc/rules/vehicles.yaml:
```diff
@@ -502,7 +502,7 @@ STNK:
 	Tooltip:
 		Name: Stealth Tank
-		Description: Long-range missile tank that can cloak. \nCan attack Aircraft. \nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks\n  Weak vs Infantry.
+		Description: Long-range missile tank that can cloak.\nCan attack Aircraft.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks\n  Weak vs Infantry.
```
They can be spotted by static defence as well.

clemty committed:
TD tooltip fixes
- mention Apache being strong vs aircraft
- mention engineers will damage buildings, not just capture them

reverted:
- Sentences are now (mostly) lowercase, end with full-stop
- Changed "defence" tab tooltip to "defense"
